.github/release.yml

```diff
@@ -1,34 +1,26 @@
 # .github/release.yml
 
 changelog:
-  exclude:
-    labels:
-      - maintenance
   categories:
-    - title: Breaking Changes
+    - title: Breaking Changes ⚠️
```

Comment:
The icon added here is rendering strangely on Chrome compared to the other icons:
![Screenshot 2024-07-25 at 1 44 03 PM](https://github.com/user-attachments/assets/233922b0-5ff1-48b6-a3cd-147c0e71eb74)
Not sure how that will affect the generated release notes though.

Reply:
weird i see that too... it was bright yellow in my text editor; maybe i'll merge with this version and see if it looks right when it gets generated, and I'll change it if it looks bad